CRITICAL - 2023-09-27 01:29:27 --> A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in APPPATH\Views\total\total42.php on line 7.
 1 APPPATH\Views\total\total42.php(7): mysqli_connect('192.168.1.178', 'root', Object(SensitiveParameterValue), 'simrs')
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\total\\total42.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('total/total42', [], true)
 5 APPPATH\Views\kelas3.php(3487): view('total/total42')
 6 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\kelas3.php')
 7 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('kelas3', [], true)
 9 APPPATH\Views\detailpasien3.php(72): view('kelas3')
10 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\detailpasien3.php')
11 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
12 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('detailpasien3', [], true)
13 APPPATH\Controllers\DetailPasienController.php(4412): view('detailpasien3', [...])
14 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
15 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
16 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
18 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\public\\index.php')
CRITICAL - 2023-09-27 01:30:00 --> A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in APPPATH\Views\total\total49.php on line 8.
 1 APPPATH\Views\total\total49.php(8): mysqli_connect('192.168.1.178', 'root', Object(SensitiveParameterValue), 'simrs')
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\total\\total49.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('total/total49', [], true)
 5 APPPATH\Views\kelas3.php(4042): view('total/total49')
 6 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\kelas3.php')
 7 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('kelas3', [], true)
 9 APPPATH\Views\detailpasien3.php(72): view('kelas3')
10 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\detailpasien3.php')
11 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
12 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('detailpasien3', [], true)
13 APPPATH\Controllers\DetailPasienController.php(4412): view('detailpasien3', [...])
14 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
15 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
16 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
18 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\public\\index.php')
CRITICAL - 2023-09-27 01:43:34 --> A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in APPPATH\Views\total\total31.php on line 8.
 1 APPPATH\Views\total\total31.php(8): mysqli_connect('192.168.1.178', 'root', Object(SensitiveParameterValue), 'simrs')
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\total\\total31.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('total/total31', [], true)
 5 APPPATH\Views\kelas3.php(2611): view('total/total31')
 6 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\kelas3.php')
 7 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('kelas3', [], true)
 9 APPPATH\Views\detailpasien3.php(72): view('kelas3')
10 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\detailpasien3.php')
11 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
12 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('detailpasien3', [], true)
13 APPPATH\Controllers\DetailPasienController.php(4412): view('detailpasien3', [...])
14 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
15 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
16 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
18 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\public\\index.php')
ERROR - 2023-09-27 01:44:06 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('192.168.1.178', 'root', Object(SensitiveParameterValue), 'simrs', 3306, '', 0)
#1 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\app\Models\DetailPasienModel.php(15): CodeIgniter\BaseModel->first()
#7 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\app\Controllers\DetailPasienController.php(3315): App\Models\DetailPasienModel->getDetailPasienData('0069')
#8 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
#9 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
#10 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\h...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\app\Models\DetailPasienModel.php(15): CodeIgniter\BaseModel->first()
#6 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\app\Controllers\DetailPasienController.php(3315): App\Models\DetailPasienModel->getDetailPasienData('0069')
#7 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
#8 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
#9 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\h...')
#12 {main}
CRITICAL - 2023-09-27 01:44:06 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `t_clinicalpathway`
WHERE `NOMR` = :NOMR:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DetailPasienModel.php(15): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\DetailPasienController.php(3315): App\Models\DetailPasienModel->getDetailPasienData('0069')
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\public\\index.php')
CRITICAL - 2023-09-27 02:21:05 --> A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in APPPATH\Views\total\total17.php on line 8.
 1 APPPATH\Views\total\total17.php(8): mysqli_connect('192.168.1.178', 'root', Object(SensitiveParameterValue), 'simrs')
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\total\\total17.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('total/total17', [], true)
 5 APPPATH\Views\kelas3.php(1500): view('total/total17')
 6 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\kelas3.php')
 7 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('kelas3', [], true)
 9 APPPATH\Views\detailpasien3.php(72): view('kelas3')
10 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\detailpasien3.php')
11 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
12 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('detailpasien3', [], true)
13 APPPATH\Controllers\DetailPasienController.php(4412): view('detailpasien3', [...])
14 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
15 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
16 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
18 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\public\\index.php')
ERROR - 2023-09-27 02:21:26 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('192.168.1.178', 'root', Object(SensitiveParameterValue), 'simrs', 3306, '', 0)
#1 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\app\Models\DetailPasienModel.php(15): CodeIgniter\BaseModel->first()
#7 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\app\Controllers\DetailPasienController.php(3315): App\Models\DetailPasienModel->getDetailPasienData('0069')
#8 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
#9 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
#10 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\h...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\app\Models\DetailPasienModel.php(15): CodeIgniter\BaseModel->first()
#6 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\app\Controllers\DetailPasienController.php(3315): App\Models\DetailPasienModel->getDetailPasienData('0069')
#7 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
#8 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
#9 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp8.2.4\htdocs\Clinicalpathway_Ci4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\h...')
#12 {main}
CRITICAL - 2023-09-27 02:21:26 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `t_clinicalpathway`
WHERE `NOMR` = :NOMR:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DetailPasienModel.php(15): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\DetailPasienController.php(3315): App\Models\DetailPasienModel->getDetailPasienData('0069')
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\public\\index.php')
CRITICAL - 2023-09-27 02:22:01 --> A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in APPPATH\Views\total\total3.php on line 8.
 1 APPPATH\Views\total\total3.php(8): mysqli_connect('192.168.1.178', 'root', Object(SensitiveParameterValue), 'simrs')
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\total\\total3.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('total/total3', [], true)
 5 APPPATH\Views\kelas3.php(330): view('total/total3')
 6 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\kelas3.php')
 7 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('kelas3', [], true)
 9 APPPATH\Views\detailpasien3.php(72): view('kelas3')
10 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\detailpasien3.php')
11 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
12 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('detailpasien3', [], true)
13 APPPATH\Controllers\DetailPasienController.php(4412): view('detailpasien3', [...])
14 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
15 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
16 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
18 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\public\\index.php')
CRITICAL - 2023-09-27 02:46:41 --> A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in APPPATH\Views\total\total18.php on line 8.
 1 APPPATH\Views\total\total18.php(8): mysqli_connect('192.168.1.178', 'root', Object(SensitiveParameterValue), 'simrs')
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\total\\total18.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('total/total18', [], true)
 5 APPPATH\Views\kelas3.php(1579): view('total/total18')
 6 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\kelas3.php')
 7 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('kelas3', [], true)
 9 APPPATH\Views\detailpasien3.php(72): view('kelas3')
10 SYSTEMPATH\View\View.php(224): include('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\app\\Views\\detailpasien3.php')
11 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
12 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('detailpasien3', [], true)
13 APPPATH\Controllers\DetailPasienController.php(4412): view('detailpasien3', [...])
14 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\DetailPasienController->kelas3()
15 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DetailPasienController))
16 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
18 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp8.2.4\\htdocs\\Clinicalpathway_Ci4\\public\\index.php')
